boxes/__init__.py

```diff
@@ -1278,6 +1301,38 @@ def flatHole(self, x, y, r=None, d=None, w=None, rel_w=0.75, angle=0):
             self.edge(2*r*math.sin(math.radians(a)))
             self.corner(-a)
 
+    @restore
+    @holeCol
+    def mountingHole(self, x, y, d_shaft=0.0, d_head=0.0, angle=0, tabs=0):
+        """
+        Draw a pear shaped mounting hole for sliding over a screw head. Total height = 1.5* d_shaft + d_head
+
+        :param x: position
+        :param y: postion
+        :param d_shaft: diameter of the screw shaft
+        :param d_head: diameter of the screw head
+        :param angle: rotation angle of the hole
+
+        """
+        
+        if d_shaft < 2*self.burn:
+            d_shaft = 2 * self.burn + 1E-9
+        rs = d_shaft/2
+        rh = d_head/2
+        
+        if not d_head or d_head < 2* self.burn:
+            self.hole(x,y,d=d_shaft,tabs=tabs)
+            return
+
```

Comment:
This makes the hole rotate around the top point instead of the center when modifying the angle. Use two .moveTo calls. The first to move to the position of the hole center pointing in the right direction and then move out to the start of the line. That way the radius moving to the line points in the right direction.